### DISCUSSION TITLE
Fahrräder als neues Gut

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#42] [Ideas]
Wäre es möglich, Fahrräder als mögliche Ladung einführen, die zusätzlich zu Fahrgästen als flexible (`0`) Beladung funktioniert?

**Comment by marhei** [comment]
Ja ist möglich ich muss aber dafür erstmal etwas bei der Berechnung der "Fahrgäste" bzw. zukünftig Fahrräder anpassen.

**Comment by marhei** [comment]
So, mit dem exchangeFactor wäre das jetzt möglich. Siehe Readme und Schema

**Comment by C1710** [comment]
Mir fällt gerade auf, dass bei den meisten (Regional)zügen ja gar nicht dabeisteht, wie viele Fahrräder da rein passen 😵‍💫
